packages/dart/noports_core/lib/src/srv/srv_impl.dart

```diff
@@ -266,45 +284,53 @@ class SrvImplInline implements Srv<SSHSocket> {
 /// - Wrap the StreamSink with encrypter
 /// - Wrap the Stream with decrypter
 class WrappedSSHSocket implements SSHSocket {
```

Comment:
various changes here all for the same reason - we don't just have a socket any more, we have a Stream and an IOSink